src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1692,7 +1692,7 @@ public SyntaxNode FindInnermostCommonExecutableBlock(IEnumerable<SyntaxNode> nod
             => nodes.FindInnermostCommonNode(node => IsExecutableBlock(node));
 
         public bool IsStatementContainer(SyntaxNode node)
-            => IsExecutableBlock(node) || node.IsEmbeddedStatementOwner();
+            => IsExecutableBlock(node) || node.IsEmbeddedStatementOwner() || node.IsKind(SyntaxKind.GlobalStatement);
```

Comment:
Are any tests affected by this change? It feels like it would make more sense to introduce this change at the same time as tests which require the change. That said, I don't think there is a need to block this PR over it.

Reply:
Not at the moment. During manual IDE test pass, I ran into asserts in IDE layer without this change. I am sure, once we add IDE tests, this code will be covered by them. 

---
In reply to: [426787734](https://github.com/dotnet/roslyn/pull/44305#discussion_r426787734) [](ancestors = 426787734)